adam/learner/object_recognizer.py

```diff
@@ -319,7 +319,7 @@ def match_objects(
                     matched_object_node, graph_to_return, pattern_match
                 )
 
-        candidate_object_subgraphs = extract_candidate_objects(perception_graph)
+        candidate_object_subgraphs = extract_candidate_objects(perception_graph, True)
```

Comment:
What is this `True` param doing? As a readability & future use note, boolean params should be keyword only.